bodhi/server/scripts/approve_testing.py

```diff
@@ -110,18 +111,43 @@ def main(argv=sys.argv):
 
                 if update.autotime and update.days_in_testing >= update.stable_days:
                     print(f"Automatically marking {update.alias} as stable")
-                    update.set_request(db=db, action=UpdateRequest.stable, username="bodhi")
+                    # For now only rawhide update can be created using side tag
+                    # Do not ad the release.pending_stable_tag if the update
```

Comment:
ad -> add

Reply:
I'm also not seeing where we're checking that the update was not created from a side tag

Reply:
At the update creation we check that only update against a release that is not composed by bodhi can be created by side tag. So here it is enough to check that the release is composed by bodhi or not.

Reply:
Is it worth mentioning this in the comment then? (My question was triggered by it)